### PR TITLE
feat(devices): added device lookup by session

### PR DESCRIPTION
**Pull request by dannycoates**
Supports https://github.com/mozilla/fxa-auth-server/pull/1166

### CODE REVIEW

File: lib/db/mem.js [hunk 1]
```diff
@@ -380,6 +380,27 @@ module.exports = function (log, error) {
       )
   }
 
+  Memory.prototype.sessionDevice = function (id) {
+    id = id.toString('hex')
+
+    if ( !sessionTokens[id] ) {
```
Comment (vladikoff) [inline lib/db/mem.js]: nit: extra spacing is not like any other style in this file 

see above ` if (! account.devices[deviceKey]) {` for instance

**Comment by vladikoff** [comment]
Also needs docs in https://github.com/mozilla/fxa-auth-db-mysql/blob/master/docs/API.md#db-api

**Comment by rfk** [comment]
assigning to @dannycoates to triage the appropriate next action on this

File: lib/db/mem.js (hunk 1 above)
Comment (dannycoates) [inline lib/db/mem.js]: its exactly like the style from the line i copied it from :laughing:

**Comment by dannycoates** [comment]
Closing this in favor of #125

**Pull request closed without merging**